jscomp/bsb/templates/react/src/index.html

```diff
@@ -10,6 +10,6 @@
   Component 2:
   <div id="index2"></div>
 
-  <script src="../build/Index.js"></script>
+  <script src="Index.js"></script>
```

Comment:
Will this change prevent the project from running without webpack-dev-server? If so, it might not be a big deal, but it might make the dev server not quite as optional.

Reply:
Got it! My bad—I totally missed that part.